### PR TITLE
MS-77: Interviews - CRUD

### PR DESCRIPTION
**Pull request by valeferaudo**

### CODE REVIEW

File: src/Components/Interviews/Form/index.js [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+      });
+    fetch(`${process.env.REACT_APP_API}/positions`)
+      .then((response) => {
+        if (response.status === 200 || response.status === 201) return response.json();
+        throw new Error(`HTTP ${response.status}`);
+      })
+      .then((response) => {
+        setPositions(response);
+      });
+    if (id !== null) {
```
Comment (damianalvarez) [inline src/Components/Interviews/Form/index.js:48]: @valeferaudo I think that this `if (id) {` is going to work as expected as well

File: src/Components/Interviews/Form/index.js [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+      } else {
+        setIsError({ ...error, [key]: false });
+      }
+    }
+    handleSubmit(newInterview);
+    handleShowModal();
+  };
+
+  return (
+    <div>
```
Comment (damianalvarez) [inline src/Components/Interviews/Form/index.js:88]: @valeferaudo what about if we get rid of this `div`? In that case, we'll be returning one element (form) as well

File: src/Components/Interviews/index.js [hunk 3]
```diff
@@ -1,9 +1,196 @@
...
+      body: JSON.stringify(interview)
+    })
+      .then((response) => {
+        if (response.status === 200 || response.status === 201) return response.json();
+        throw new Error(`HTTP ${response.status}`);
+      })
+      .then((response) => {
+        setShowMessage(true);
+        setMessageType('success');
+        setMessage('Candidate updated');
```
Comment (sofpag) [inline src/Components/Interviews/index.js:88]: you should change candidate for interview

**Review by damianalvarez — CHANGES_REQUESTED** [review]

File: src/Components/Interviews/interviews.module.css [hunk 4]
```diff
@@ -1,6 +1,44 @@
...
   flex: 1;
   display: flex;
+  flex-direction: column;
   justify-content: center;
   align-items: center;
-}
\ No newline at end of file
+}
+
+
```
Comment (sofpag) [inline src/Components/Interviews/interviews.module.css:9]: strange that this space didn't throw error

File: src/Components/Shared/Confirmation/confirmation.module.css [hunk 5]
```diff
@@ -0,0 +1,6 @@
+.container {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+  }
```
Comment (sofpag) [inline src/Components/Shared/Confirmation/confirmation.module.css:6]: you need to format this file

File: src/Components/Shared/Modal/index.js [hunk 6]
```diff
@@ -1,25 +1,35 @@
 import React from 'react';
 import styles from './modal.module.css';
 import Button from '../Button';
+import InterviewForm from '../../Interviews/Form/index';
```
Comment (sofpag) [inline src/Components/Shared/Modal/index.js:4]: you should delete /index

**Review by sofpag — APPROVED** [review]

**Review by damianalvarez — APPROVED** [review]

**Pull request merged**